### DISCUSSION TITLE
2 dexcom sensors simultaneously

### DISCUSSION THREAD

**Discussion by usernamebeep** [NightscoutFoundation/xDrip#1969] [Ideas]
Is it possible to use 2 dexcom (or any other brand) sensors simultaneously?
Dexcom takes 2 hours to warm up which means, no data for 2 hours. If one could start second sensor while still using first one, there won't any data gap.
Current data would still be shown from first sensor but after warm up, second sensor would be main sensor.

**Comment by Navid200** [comment]
If you have 2 transmitters, you can do that.  
But, you cannot do it with only one xDrip app.  You will need two.  
But, the requirement for 2 transmitters would be the main reason not to consider this for most people.
Are you willing to double your cost for transmitters?

**Comment by usernamebeep** [comment]
That's a good point that one would need 2 transmitters. Did not thought of that since I get my sensors/transmitters for free.

How do I install xdrip twice? I guess data is not synced between apps. Would be nice if this feature was built into xdrip.

**Reply by Navid200** [reply]
You will need a variant in order to have more than one instance of xDrip:
https://navid200.github.io/xDrip/docs/Variants.html
